Examples/ObjectiveC/LineStyleLayerExample.m

```diff
@@ -16,7 +16,7 @@ - (void)viewDidLoad {
     self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
 
     [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(45.5076, -122.6736)
-			    zoomLevel:11
+                zoomLevel:11
                  animated:NO];
```

Comment:
Nit: align all three colons.

Reply:
You can select all three lines of this method call and press ⌃I to reindent them correctly.